src/content/docs/en/recipes/add-yaml-support.mdx

```diff
@@ -34,3 +34,15 @@ Astro builds on top of Vite, and supports both Vite and Rollup plugins. This rec
     ```js
     import yml from './data.yml';
     ```
+
+    :::note
+    While you can now import YAML data in your Astro project, your editor will not provide types for the imported data. To add types, create a `*.d.ts` file in the `src` directory of your project and add the following:
+    ```ts
+    // Specify the file extension you want to import
+    declare module "*.yml" {
+      const value: any; // Add better type definitions here if desired
```

Comment:
```suggestion
      const value: any; // Add extra type definitions here if desired
```
Can we go for a different word for "better"?  :sweat_smile: I tried "extra" here, but you could say something like "additional" or "Add any desired type definitions here"

Reply:
Could removing the word entirely work? Since `any` isn't really a type, and is just a placeholder.
```suggestion
      const value: any; // Add type definitions here if desired
```

If not "additional" sounds like it works

```suggestion
      const value: any; // Add additional type definitions here if desired
```